src/overlays/actors/ovl_En_Syateki_Niw/z_en_syateki_niw.c

```diff
@@ -477,93 +478,93 @@ void func_80B12460(EnSyatekiNiw* this, PlayState* play) {
                        RAD_TO_BINANG(Math_FAtan2F(player->actor.world.pos.x - this->actor.world.pos.x,
                                                   player->actor.world.pos.z - this->actor.world.pos.z)) +
                            phi_f16,
-                       5, this->unk_2C8.y, 0);
-    Math_ApproachF(&this->unk_2C8.y, 3000.0f, 1.0f, 500.0f);
-    if (this->unk_296 == 2) {
-        this->unk_256 = 10;
-        this->unk_254 = this->unk_256;
+                       5, this->posRotStep.y, 0);
+    Math_ApproachF(&this->posRotStep.y, 3000.0f, 1.0f, 500.0f);
+    if (this->archeryAnimationType == 2) {
+        this->timer1 = 10;
+        this->peckTimer = this->timer1;
```

Comment:
```suggestion
        this->peckTimer = this->timer1 = 10;
```
this change typically matches (and it matches here too)